gateapi/api/swagger.yaml

```diff
@@ -6366,7 +6445,7 @@ paths:
         schema:
           type: "array"
           items:
-            $ref: "#/definitions/MapÂ«string,stringÂ»"
```

Comment:
I'm not sure where these odd characters come from. Swaggergen?

Reply:
Nice that they're disappearing...